tools/validate-json-schemas.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+if ! command -v ajv &> /dev/null
+then
+    echo "The 'ajv' command is required, please install the 'ajv-cli' package"
```

Comment:
What kind of package is this? Can we give contributors more instruction about how to do this, either in the readme or here?

Reply:
1f9ce833f67b9761f01603f7de01638596a83858

I decided to add a link to the GitHub repo in the script itself as IMO it's a detail that is only relevant to this one script. I have also referenced this script in CONTRIBUTING.md.

What do you think? I'm open to feedback :)